packages/zowe-explorer/README.md

```diff
@@ -297,22 +297,19 @@ Your data set member is uploaded.
    ![Parameters](/docs/images/ZE-set-params.gif?raw=true "Parameters")
    <br /><br />
 
-### Delete a data set member and a data set
+### Delete data sets and data set members
 
 1. Navigate to the explorer tree.
 2. Open the **DATA SETS** bar.
-3. Open the profile and a data set containing the member.
-4. Right-click the data set member that you want to delete and select **Delete Member**.
+3. Select one or more data sets and/or data set members
+   (to select more than one item for deletion, hold **CTRL/CMD** key while clicking the items)
+4. Press **DELETE** on your keyboard
 5. Confirm the deletion by clicking **Delete** in the drop-down menu.
+6. You can also delete a data set or member from the context menu.
+   Right-click on the item and select **Delete Data Set** or **Delete Member** from the context menu.
```

Comment:
I think point 6 should go with point 4 since the confirmation to delete will happen after either the keyboard or context menu action.

Reply:
Changed this...could you review again please, when you get a chance?